src/Clap-Core.package/ClapAlternative.class/README.md

```diff
@@ -0,0 +1,19 @@
+I am a class which represents all the possible ways to execute a command.
+
+For example,  imagine you have two ways to execute the command eval with the help flag, the first one is "eval -h <exp>" and the second one is "eval --help <exp>".You create a ClapCommand for the first: 
+	c1 := (ClapCommand withName: 'eval')
+		addFlag: ClapFlag withName: 'h';
+		...
+		
+And the second:  
```

Comment:
No, you create a single `ClapCommand` and add an alternative if there is a choice between parameters. It's not even useful to do it for flags since they will already recognize both the short and long versions.